Added note to Update Player Endpoint (#845)
freyacodes/Lavalink: IMPLEMENTATION.md

| filters?        | [Filters](#filters) object         | The new filters to apply. This will override all previously applied filters   |                   
| voice?          | [Voice State](#voice-state) object | Information required for connecting to Discord, without `connected` or `ping` |

> **Notes**:
>  - that `encodedTrack` and `identifier` are mutually exclusive.
>  - `sessionId` in the route should be value that you got from the [ready](#ready-op) op.

Reviewer: ```suggestion
> **Note**
> - `encodedTrack` and `identifier` are mutually exclusive.
> - `sessionId` in the path should be the value from the [ready op](#ready-op).
```
